### DISCUSSION TITLE
Upgrade video quality (e.g. SD -> 1080p)

### DISCUSSION THREAD

**Discussion by epheterson** [sct/overseerr#2638] [Feature Requests]
With a large library, some of the older content does exist, but is in a low quality. It'd be great if users could utilize Overseer to request Sonarr or Radarr upgrade the quality of an unmonitored file that otherwise will not upgrade on its own.

Ideally the user would only have to click one button, and Overseer would instruct Radarr/Sonarr to upgrade to default Overseer quality profile. The button should conditionally be displayed if the highest quality in Plex is less than the default Overseer quality profile.

**Comment by Pepeanuts** [comment]
+1

i'm looking for this feature too

**Comment by KMDPyro** [comment]
+1  I just started using overseerr and it's always been a sore spot with other requesting methods, poor ability to request a new quality. Sometimes the old version you had was just low quality because you didn't want to use up as much space. Then after a disk upgrade you want to go back and upgrade some things. Sometimes you might also want to downgrade. I recently got a 4K title that was worth watching but not worth having in 4K and had to delete and re-request at 720p or 1080.

**Comment by abyssalsmoke** [comment]
this would be awesome.  I have lots of 1080 films I would like to request upgrade to 4k but currently have to go into sonarr if it already exists in plex.

**Comment by Laiteux** [comment]
Same here, would be amazing because I have friends who have 4K monitors and it would be nice for them to be able to upgrade the quality profile of a movie, if someone else already has downloaded it in 1080p before.

@TheCatLady not sure if I should ping, only doing so because this thread is lacking a reply from the team (: Really sorry if not!

**Reply by sct** [reply]
We already support separate requests for 1080p and 4K. Are you using this?

**Reply by Madbeefer** [reply]
I don't think the requests allow for something that already exists in the library to be requested for like a 480 to 1080 upgrade.  You could request in 4k if it already exists, but that doesn't allow for a non 4k upgrade.

**Comment by daneger** [comment]
I know this thread is old but this is the exact feature I came here to find. Letting a user be able to request a video that is already in the library at 480p but at a higher bit rate would be great. Admittedly not a deal breaker as this is the best request software in my opinion, but a would be nice feature.

**Reply by Madbeefer** [reply]
Well this thread has been dead for over a year has this feature been deemed not something that will be added?

**Reply by Alex-Lee-Myers** [reply]
And now we're nearing 2025 and still nothing I can find on this being a feature to be added. Ombi can do it. I don't understand why this application couldn't.

**Reply by funkybside** [reply]
just got here myself for the same reason.    Once you have your stack up and playing nice with overseer sitting on top of it all, it's very tedious to go deal with this at deeper layers.

**Comment by sage2050** [comment]
I just merged my libraries only to find I can't do this

**Comment by bugalou** [comment]
For those who care after much pain and suffering I managed to get an Upgrade Quality option added to the Report Issues system. Its very basic - it just logs an issue asking to upgrade the video quality to HD, UHD, or AI Upscale or to upgrade the audio to surround sound.  It requires manual intervention by the admin to upgrade the media on the back end. This is desired as in my case, I don't want people doing  uncontrolled upgrades on my server, but also do not want to approve requests manually.  I am not a typescript dev so my code is probably awful so feel free to clean it up. I did not put in a pull request for this either since I am not confident in my code (it is working in my testing though) and I am not sure if the devs here even want this since there is an advanced request function. Anyways to implement in docker you can do this:

`git clone https://github.com/bugalou/docker-overseerr.git`
`cd docker-overseerr`
`docker build -t overseerrbug .`


create the local folder you want to use to store the db and config and cd in to it (or change the . in the path below to your folder then:

`docker run -d -e LOG_LEVEL=debug -e TZ=America/Chicago -e PORT=5056 -p 5056:5056 -v .:/config --restart unless-stopped --name overseerrbug overseerrbug
`

I opted to change the port to 5056 so I could run this side by side with my standard overseerr instance for a while to be sure all is well. If you want to change this you are likely going to need to modify the Dockerfile from the git clone command above and change the port set in the docker run command.  I cannot stress enough how much I am winging it here, I know just a little bit about docker and ziltch about typescript, though I know other languages so I have a general concept. If someone knows how to do things better, do add to/correct this.  

Hope this helps someone else because I spent far too much time on it today. 😫
screens:

![Screenshot_55](https://github.com/user-attachments/assets/b88a2c19-29dc-4785-87ac-006fbeef8241)
![Screenshot_56](https://github.com/user-attachments/assets/7301302c-4278-495f-b08f-57baac81a5b3)
![Screenshot_57](https://github.com/user-attachments/assets/ce75d438-6fa9-4b54-992b-71cccd2a6291)
